Automattic-Tracks-iOS.podspec

```diff
@@ -32,24 +32,5 @@ Pod::Spec.new do |spec|
   spec.dependency 'CocoaLumberjack', '~> 3'
   spec.dependency 'Reachability', '~> 3'
   spec.dependency 'Sentry', '~>4'
-  # This is a fork of Sodium that incorporates a fix to make it validate with
-  # the Xcode 12.0 toolchain and CocoaPods 1.9.3
-  spec.dependency 'Sodium-Fork', '0.8.2'
-
-  # Xcode 12 changed the way apps are built because of the upcoming support for
-  # Apple Silicon. We need explicitly exclude the arm64 architecture.
-  # More info at
-  # https://stackoverflow.com/questions/63607158/xcode-12-building-for-ios-simulator-but-linking-in-object-file-built-for-ios
-  spec.pod_target_xcconfig = {
-    'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64',
-  }
-
-  # Notice that a CocoaPods contributor discourages the use of
-  # `user_target_xcconfig`, but in this case also agrees that there might not be
-  # a better approach.
-  # See
-  # https://github.com/CocoaPods/CocoaPods/issues/10065#issuecomment-701055569
-  spec.user_target_xcconfig = {
-    'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64',
-  }
+  spec.dependency 'Sodium', '0.9.1'
```

Comment:
Excellent! I missed the 0.9.1 update with the arm64 update 🎉

Reply:
This should be `~> 0.9` – the `.lock` file will ensure we never downgrade (unless explicitly asked to), and this way other dependencies that may use sodium won't break.

Reply:
Since `0.9.0` won't work for us we should in theory specify `0.9.1` because a podspec can't enforce a hash like a lock file can. If someone doesn't update the Cocoapods repo and 0.9.0 is the latest version, then that's the one that'll get installed. If they follow semantic versioning, it's effectively doing the same thing as `~> 0.9` because there is no version `1.10.x`.

Reply:
(`0.9.1` has the arm64 fix)

Reply:
Ah – you are correct – the `.lock` file only applies to a pre-existing pod file 🤦 

In this case it would be preferable to use `>= 0.9.1` IMHO – no lower, but allow higher versions if they're used in other dependencies to avoid conflicts.

Reply:
Ooh, yes, that's a better one!

Reply:
> In this case it would be preferable to use >= 0.9.1 IMHO – no lower, but allow higher versions if they're used in other dependencies to avoid conflicts.

+1 👍